### ISSUE TITLE
Error executing jobs:  error linkData: symlink /pfs/.scratch/import /pfs/import: file exists

### ISSUE THREAD

**Issue by shimpel** [pachyderm/pachyderm#3237]
**What happened?**: 
I have create a repo which contains a single sql file. This contains a sql statement.
The pipeline conists of an own container which contains a python file to execute the script, query the database and download the result from a minio object store.
The relevant code looks  like this:

```
def execute_sql(cur, sql):
    # execute sql
    cur.execute(open(sql, "r").read())
    return cur.fetchall()


def download_file_minio(minio_client, filename):
    try:
        data = minio_client.get_object('test', filename)
        with open(os.path.join(".", filename), 'wb') as file_data:
            for d in data.stream(32*1024):
                file_data.write(d)
            for dirpath, dirs, files in os.walk("/pfs/out"):
                if filename not in files:
                    shutil.copy2(os.path.join(".", filename),
                                 os.path.join("/pfs/out", filename))
    except ResponseError as err:
        print(err)


def main():
    print('DB')
    cur = connect_database()
    print('minio')
    minio_client = connect_mino()
    for dirpath, dirs, files in os.walk("/pfs/import"):
        for file in files:
            if '.sql' in file:
                for row in execute_sql(cur, os.path.join(dirpath, file)):
                    print(row[2])
                    download_file_minio(minio_client, row[2])


main()
```

When the pipeline runs the job fails with error: 
"failed to process datum with error: error linkData: symlink /pfs/.scratch/ccd19a7f74f54b09ac832ff6eee9eb11/import /pfs/import: file exists"

The pipeline spec is:
```json
{
  "pipeline": {
    "name": "test13"
  },
  "transform": {
    "image": "XXX:5000/import",
    "cmd": [
      "/bin/sh",
      "-c",
      "/usr/bin/python3 import.py"
    ]
  },
  "parallelismSpec": {
    "constant": "1"
  },
  "egress": {

  },
  "outputBranch": "master",
  "resourceRequests": {

  },
  "resourceLimits": {

  },
  "input": {
    "atom": {
      "name": "import",
      "repo": "import",
      "branch": "master",
      "glob": "/"
    }
  },
  "cacheSize": "64M",
  "enableStats": true,
  "maxQueueSize": "1",
   "salt": "97b06e8e86e443229c1e32fa92b9a10f"
}
```

**What you expected to happen?**:
Job can be run successful.

**How to reproduce it (as minimally and precisely as possible)?**:
See the description for the code.

**Anything else we need to know?**:
The script runs on my local machine or in the docker container (with adjustet file path) without any problems.
If I run kubectl exec <pod> ls /pfs there are 2 folders (import and out). Import just contains my sql file. Out contains all downloaded files. So everything seems fine for me.
**Environment?**:
- Kubernetes version (use `kubectl version`): Client Version: version.Info{Major:"1", Minor:"12", GitVersion:"v1.12.1", GitCommit:"4ed3216f3ec431b140b1d899130a69fc671678f4", GitTreeState:"clean", BuildDate:"2018-10-05T16:46:06Z", GoVersion:"go1.10.4", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"12", GitVersion:"v1.12.2", GitCommit:"17c77c7898218073f14c8d573582e8d2313dc740", GitTreeState:"clean", BuildDate:"2018-10-24T06:43:59Z", GoVersion:"go1.10.4", Compiler:"gc", Platform:"linux/amd64"}

- Pachyderm CLI version (use `pachctl version`): pachctl             1.7.11              
pachd               1.7.11  
- Cloud provider (e.g. aws, azure, gke) or local deployment (e.g. minikube vs dockerized k8s): kubeadm
- OS (e.g. from /etc/os-release): Ubuntu 18.04
- Others:

**Comment by JoeyZwicker** [comment]
Thanks @shimpel, we'll double check, but I think it should be resolved in 1.8.0

**Comment by shimpel** [comment]
Hi @JoeyZwicker. After the update to 1.8.0 this doesn't occur any longer. 
I will close this issue.
